### PR TITLE
Isolate Logging Services

### PR DESCRIPTION
**Pull request by evanugarte**
We now have a seperate server to add and fetch logs for Errors and Sign activity. It's part of the effort to make our architecture to look like this:
image

### CODE REVIEW

File: api/util/logging-helpers.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const { LOGGING_API_URL } = require('../config/config');
+const axios = require('axios');
+
+/**
+ * Add a new error by calling on the ErrorLog API.
+ * @param {Object} newError - The error that is to be added
```
Comment (hsorathia) [inline api/util/logging-helpers.js:6]: Is this a parameter here?

File: api/util/logging-helpers.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * @param {(string|undefined)} errorToAdd.userEmail - The email of the user
+ *                                                  who has sent this error
+ * @param {(string|undefined)} errorToAdd.errorTime- The time the error occured
+ * @param {string} errorToAdd.apiEndpoint - The location of the error
+ * @param {string} errorToAdd.errordescription - The description of the error
+ */
```
Comment (hsorathia) [inline api/util/logging-helpers.js:12]: To have consistency with the comments for the function below, maybe add an `@return` comment for errorSaved to describe what it does

File: src/Pages/MembershipApplication/MembershipPlan.js [hunk 3]
```diff
@@ -162,6 +163,11 @@ class MembershipPlan extends Component {
             </div>
           </div>
         </Container>
+        <Button onClick={() => {
+          sendVerificationEmail('evanuxd@gmail.com', 'evan');
```
Comment (hsorathia) [inline src/Pages/MembershipApplication/MembershipPlan.js:167]: nice email xd

File: api/util/logging-helpers.js (hunk 1 above)
Comment (law-nguyen) [reply api/util/logging-helpers.js:6]: doesn't seem to be to used in this function

Comment (evanugarte) [reply api/util/logging-helpers.js:6]: uh oh!!!! got me

File: api/util/logging-helpers.js (hunk 2 above)
Comment (evanugarte) [reply api/util/logging-helpers.js:12]: yes thank you for catching this. adding it now.

File: src/Pages/MembershipApplication/MembershipPlan.js (hunk 3 above)
Comment (evanugarte) [reply src/Pages/MembershipApplication/MembershipPlan.js:167]: whoops forgot to remove this, thanks again

File: api/logging/SignLog.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  const newSign = new SignLog({
+    signText: req.body.signText,
+    firstName: req.body.firstName,
+    email: req.body.email,
+    timeOfPosting: req.body.timeOfPosting
+  });
+
+  newSign.save(function(error) {
+    if (error) {
+      console.log('weeee dead', error);
```
Comment (evanugarte) [inline api/logging/SignLog.js:16]: shoulda removed this too

**Pull request merged**